src/foam/u2/view/PasswordView.js

```diff
@@ -68,24 +69,34 @@ foam.CLASS({
       name: 'type',
       value: 'password'
     },
-    'inputElement'
+    'inputElement',
+    [ 'opacity', 0.3 ]
   ],
 
   methods: [
     function initE() {
       this.SUPER();
 
-      this.addClass(this.myClass()).start().
-        start(this.TextField, {
+      this.addClass(this.myClass()).start()
+        .start(this.TextField, {
           type: this.type,
           data$: this.data$,
-          onKey: true
-        }, this.inputElement$).
-        addClass('full-width-input-password').end().
-        start('img').show(this.passwordIcon$).addClass('input-image').
-          attr('src', this.visibilityIcon$).on('click', this.visible).
-        end().
-      end();
+          onKey: true,
+
+        }, this.inputElement$)
+        .on('focus', () => this.opacity = 1)
```

Comment:
Couldn't this be done more easily with just CSS?

Reply:
I wish